client/components/phone-input/index.jsx

```diff
@@ -23,21 +23,24 @@ const PhoneInput = React.createClass( {
 		countriesList: React.PropTypes.object.isRequired
 	},
 
+	/**
+	 * Returns the country meta with default values for countries with missing metadata. Never returns null.
```

Comment:
Yes, if `selectedCountry` is falsy, we create a mock in the if branch and return that, otherwise we return what we find under `countries[ countryCode ]`